config/samples/aiml_v1beta1_pachyderm.yaml

```diff
@@ -3,6 +3,7 @@ kind: Pachyderm
 metadata:
   name: pachyderm-sample
 spec:
+  version: "v2.0.3"
```

Comment:
Is there a reason you want to add the version here? I would prefer to leaving it out and let the operator dynamically choose the most recent supported version of Pachyderm.

Adding the version would require the example to be updated with each new release.

Reply:
Not specifying a version currently results in a panic by the operator. This works around that. Adding code to get the latest version is a good idea, but from what I understand, not currently present in the operator

Reply:
Sorry for the long delay to get back to you. I have not seen this issue myself. However,  the [webhook](https://github.com/pachyderm/openshift-operator/blob/678f7815beca5a29228254c9b2b23f57abd0c4e0/api/v1beta1/pachyderm_webhook.go#L61) has code to set the default version of the Pachyderm chart to use.

Reply:
I think this will apply for a small edge case where someone deploys Pachyderm from the sample resource in `config/samples`. Otherwise it should be auto-filled.